### PR TITLE
[DEX] Transaction Explorer

### PR DESCRIPTION
**Pull request by crishellco**
## Ticket
* n/a

## Changes
* Add fly-out recent transaction explorer

## Screenshots
screen shot 2018-08-30 at 10 00 23 am
screen shot 2018-08-30 at 10 00 19 am
screen shot 2018-08-30 at 10 00 07 am
screen shot 2018-08-30 at 10 00 03 am

## Code Coverage
image

### CODE REVIEW

File: src/renderer/components/SimpleTransactions.vue [hunk 1]
```diff
@@ -91,6 +100,28 @@ export default {
...
+    &.status {
+      text-align: right;
+
+      .aph-icon {
+        svg {
+          height: toRem(14px);
+
+          &.confirmed {
+            .fill {
+              fill: $green !important;
```
Comment (schaeferCode) [inline src/renderer/components/SimpleTransactions.vue]: This is interesting. I'm assuming you tried other ways to style this? There some other styling override this?

File: src/renderer/components/TransactionsSidebar.vue [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+      }
+    }
+
+    &:hover {
+      background: $light-grey;
+    }
+  }
+
+  > .content {
+    display:none;
```
Comment (schaeferCode) [inline src/renderer/components/TransactionsSidebar.vue]: Hit the spacebar once after the colon.

File: src/renderer/components/TransactionsSidebar.vue [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+      > .header {
+        h1.underlined {
+          color: $copy-night;
+        }
+      }
+    }
+  }
+}
+
+.Night {
```
Comment (schaeferCode) [inline src/renderer/components/TransactionsSidebar.vue]: Should this be capitalized?

File: src/renderer/components/SimpleTransactions.vue (hunk 1 above)
Comment (crishellco) [reply src/renderer/components/SimpleTransactions.vue]: Yeah, sucks but I tried.

File: src/renderer/components/TransactionsSidebar.vue (hunk 2 above)
Comment (crishellco) [reply src/renderer/components/TransactionsSidebar.vue]: 👍

File: src/renderer/components/TransactionsSidebar.vue (hunk 3 above)
Comment (crishellco) [reply src/renderer/components/TransactionsSidebar.vue]: No, but its how it was done when the DEX was first built. Its a tech item.

**Comment by jsolman** [comment]
Would like to have this pane show pending transactions as well in the future.

**Pull request merged**